delfin/drivers/fake_storage/__init__.py

```diff
@@ -468,41 +477,58 @@ def _get_volume_range(self, start, end):
             volume_list.append(v)
         return volume_list
 
-    def _get_random_performance(self):
+    def _get_random_performance(self, metric_list, start_time, end_time):
         def get_random_timestamp_value():
             rtv = {}
-            for i in range(MIN_PERF_VALUES, MAX_PERF_VALUES):
-                timestamp = int(float(datetime.datetime.now().timestamp()
-                                      ) * 1000)
+            timestamp = start_time
```

Comment:
is it system time?!/regional specific?

Reply:
how is this used?

Reply:
This is timestamp equivalent to UTC. since we support historic collection , this is the fake driver side emulation to support collection from provided start_time to end_time